tests/testAutopas/tests/containers/CellBlock3DTest.h

```diff
@@ -14,19 +14,23 @@ class CellBlock3DTest : public AutoPasTestBase {
  public:
   CellBlock3DTest()
       : _cells_1x1x1(_vec1, std::array<double, 3>({0.0, 0.0, 0.0}), std::array<double, 3>({10.0, 10.0, 10.0}), 10.0),
+        _cells_1x1x1_cs(_vec1_cs, std::array<double, 3>({0.0, 0.0, 0.0}), std::array<double, 3>({10.0, 10.0, 10.0}),
```

Comment:
```suggestion
        _cells_1x1x1_cs2(_vec1_cs, std::array<double, 3>({0.0, 0.0, 0.0}), std::array<double, 3>({10.0, 10.0, 10.0}),
```
Also adapt the other names